### PR TITLE
2FA Settings: Content, icons, and tooltips

### LINKED ISSUE #412: 2FA Contextual Help
*note: we may potentially need to create a Pub specific help section on the site in the future. till then, we'll handle extra sideline content in the form of contextual help flyout.
**any suggestions on the wording are welcome.

**current browser support limitation**
2fa-overlay-browserw

**supported/tested u2f keys**
2fa-overlay-supportedkeys

### LINKED ISSUE #419: 2FA Home: content & various states
I had to make minor updates on the content in Hardware security keys section to make the 'browser limitation' blurb more noticeable.  -- this is to prevent user frustration if they discover it after the fact.  

- reference mocks below indicating different states of 2FA main page + the contextual help

--------------------------

## Two-factor Authentication

Two-factor authentication (2FA) is a method of confirming your identity by using two different forms of verification when you access Brave Payments in order to increase security (Recommended for protecting your account).

**App on your phone**
Use an app on your phone to get an authentication code. You will be asked to type in this code when logging in. 

**Hardware security keys**
Security key is a small device that connects to your computer via a USB port and works with FIDO Universal 2nd Factor (U2F). You will be asked to insert and press the key instead of typing in a code. Currently, security key is supported by Google Chrome and Opera. 
*A list of security keys that have been tested

--
**Limited Browser Support** contextual help flyout
This means you will need to update to the latest Google Chrome or Opera in order to set up and log in using security keys. More browsers will start supporting the security key in the near future. We will inform you on the changes.

--
**Tested Security Keys** contextual help flyout
The following products have been tested and proven to work with our system. 
- YubiKey
- Ledger Blue
- Product 3
- Product 4
- Product 5
--------------------------

**1. 2FA Disabled**
2fa-home-a0k0

2. TOTP Enabled; U2F disabled
2fa-home-a1k0

3. TOTP disabled; U2F enabled
2fa-home-a0k1

4. Both enabled
2fa-home-a1k1

5. Contextual help for the supported browser
2fa-overlay-browser

6. Contextual help for the supported key list
2fa-overlay-supportedkeys

### PR DESCRIPTION
**Pull request by kpfefferle**
Resolves #412 
Resolves #419 

**Content and Icons:**
Content and Icons

**Browser Tooltip:**
Browser Tooltip

**Device Tooltip:**
Device Tooltip

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave-intl/publishers/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Tagged reviewers.

Reviewer Checklist:

Tests
- [ ] Adequate test coverage exists to prevent regressions

Security:
- [ ] No raw SQL -- Always prefer ActiveRecord query helpers ([more info on StackOverflow](https://stackoverflow.com/questions/41410752/rails-5-sql-injection#41452695))
- [ ] XSS is mitigated -- Avoid `html_safe` and `raw`; escape untrusted content from users and 3rd party APIs ([Rails XSS guide](https://brakemanpro.com/2017/09/08/cross-site-scripting-in-rails) and [OWASP XSS Prevention Cheat Sheet](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet))

### CODE REVIEW

File: app/views/application/_usb.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="" height="75" viewBox="0 0 73 75">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="58" height="50" viewBox="0 0 58 50">
```
Comment (mixonic) [inline app/views/application/_usb.html]: This size change will also impact the U2F authentication flow. We should confirm that looks correct before landing.

Comment (kpfefferle) [reply app/views/application/_usb.html]: I couldn't find anywhere else where this partial was being used, but if we need to we can resize via CSS in one place or another.

Comment (kpfefferle) [reply app/views/application/_usb.html]: Ah, I see it here:
https://github.com/brave-intl/publishers/blob/aa6eab1fcc7af350499f71d0878977e1f6693e06/app/views/two_factor_authentications/_u2f.html.slim#L25

Comment (kpfefferle) [reply app/views/application/_usb.html]: @mixonic I looked into this, and it seems my "new" version is actually the exact same rendered size for the icon itself, but without added padding and rotation information included in the SVG file. This seems to be an improvement in the export itself, and it still looks good when shown during auth:
USB icon during 2FA auth

**Comment by ayumi** [comment]
looks good, thanks! :parasol_on_ground:

**Pull request merged**